plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduRecordCursor.java

```diff
@@ -43,10 +43,11 @@
 
     static {
         try {
-            ROW_DATA_FIELD = RowResult.class.getDeclaredField("rowData");
+            Class<?> clazz = Class.forName("org.apache.kudu.client.RowwiseRowResult");
```

Comment:
If we use `currentRow.getSchema().getRowSize()` then we might not need this snippet right ?

Reply:
Yes after this PR is merged:
https://github.com/trinodb/trino/pull/10940/commits

This will go away.